flowplayer.desktop

```diff
@@ -6,4 +6,8 @@ Exec=flowplayer
 Name=FlowPlayer
 
 [X-Sailjail]
-Sandboxing=Disabled
+OrganizationName=sailfishos-applications
+# ApplicationName does not have to be identical to Name
+ApplicationName=flowplayer
+# Add the required permissions here
+Permissions=UserDirs;Audio;MediaIndexing;Bluetooth;Internet;RemovableMedia
```

Comment:
Do we really need MediaIndexing ? I thought it was to use `tracker. Maybe I'm wrong. I'll test next week.

Reply:
I don't think we need MediaIndexing. I believe I was just copying from the media player app? I'll remove and test.

Reply:
Done in last commit.